notebooks.go

```diff
@@ -478,14 +496,14 @@ func (s *server) NewNotebook(w http.ResponseWriter, r *http.Request) {
 	//Add Language
 	//Validate that the language format is valid (language[_territory])
 	match, err := regexp.MatchString("^[[:alpha:]]{2}(_[[:alpha:]]{2})?$", req.Language)
-		if (err != nil || !match) {
-			var errLanguageFormat = errors.New("Error: the value of KF_LANG environment variable ('" + req.Language + "') is not a valid format (e.g 'en', 'en_US', ...)")
-			s.error(w, r, errLanguageFormat)
-			return
-		}
+	if err != nil || !match {
+		var errLanguageFormat = errors.New("Error: the value of KF_LANG environment variable ('" + req.Language + "') is not a valid format (e.g 'en', 'en_US', ...)")
```

Comment:
Is there a way to just send a message back to front end and have a i18n message?

Reply:
This wasn't a part of our changes it just formatted I think